src/libm-tester/tester2sp.c

```diff
@@ -38,6 +38,10 @@
 static int isinff(float x) { return x == __builtin_inff() || x == -__builtin_inff(); }
 #endif
 
+#if defined(__FreeBSD__) 
+#define isinff(x) ((x) == (float)(1e+300) || (x) == -(float)(1e+300))
```

Comment:
this is already defined in /src/misc/common.h and it is different:

```#define isinff(x) ((x) == SLEEF_INFINITYf || (x) == -SLEEF_INFINITYf)```

Which is the correct one for freebsd?

Reply:
Both are correct.
SLEEF_INFINITYf is not defined in sleefsp.c, and so (float)(1e+300) is used instead.